lib/vehicle.rb

```diff
@@ -5,18 +5,26 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
-
+              :engine,
+              :plate_type
+  attr_accessor :registration_date,
+                :plate_type
+                
   def initialize(vehicle_details)
     @vin = vehicle_details[:vin]
     @year = vehicle_details[:year]
     @make = vehicle_details[:make]
     @model = vehicle_details[:model]
     @engine = vehicle_details[:engine]
+    @registration_date = nil
+    @plate_type = nil
   end
 
   def antique?
-    Date.today.year - @year > 25
+    Date.today.year - @year >= 25
```

Comment:
You could make your code more DRY (don't repeat yourself) by reusing the age method:
```
age >= 25
```